sdcm/version_utils.py

```diff
@@ -0,0 +1,55 @@
+import requests
```

Comment:
the dir structure should be as follows:
sdcm/utils/scylla_version.py
sdcm/utils/common.py (move and rename `utils.py`)

Reply:
changed